File: sample/src/main/java/com/leibo/repertory/main/MainActivity.java
package com.leibo.repertory.main;

import android.support.annotation.NonNull;
import android.view.View;

import com.leibo.baselib.base.BaseActivity;
import com.leibo.repertory.player.PlayerMainActivity;
import com.leibo.repertory.recycler.RecyclerPageActivity;
import com.leibo.repertory.widget.CustomViewActivity;
import com.leibo.repertory.R;

public class MainActivity extends BaseActivity {

    @Override
    protected int getLayoutResID() {
        return R.layout.activity_main;
    }

    public void onClick(@NonNull View view) {
        switch (view.getId()) {
            case R.id.btn_widget:
                startActivity(CustomViewActivity.class);
                break;
            case R.id.btn_recycler:
                startActivity(RecyclerPageActivity.class);
                break;
            case R.id.btn_player:
                startActivity(PlayerMainActivity.class);
                break;
        }
    }
}

File: sample/src/main/java/com/leibo/repertory/main/MainActivity.kt
package com.leibo.repertory.main

import android.view.View
import com.leibo.baselib.base.basic.BaseActivity
import com.leibo.repertory.R
import com.leibo.repertory.image.ImageTestActivity
import com.leibo.repertory.player.PlayerMainActivity
import com.leibo.repertory.recycler.RecyclerPageActivity
import com.leibo.repertory.widget.CustomViewActivity

class MainActivity : BaseActivity() {

    override val layoutResID: Int
        get() = R.layout.activity_main

    override fun onClick(view: View) {
        when (view.id) {
            R.id.btn_widget -> startActivity(CustomViewActivity::class.java)
            R.id.btn_recycler -> startActivity(RecyclerPageActivity::class.java)
            R.id.btn_player -> startActivity(PlayerMainActivity::class.java)
            R.id.btn_image -> startActivity(ImageTestActivity::class.java)
        }
    }

}
